server/src/test/java/com/datastrato/gravitino/server/web/mapper/TestJsonMappingExceptionMapper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.server.web.mapper;
+
+import com.datastrato.gravitino.dto.responses.ErrorResponse;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import javax.ws.rs.core.Response;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class TestJsonMappingExceptionMapper {
+  private final JsonMappingExceptionMapper jsonMappingExceptionMapper =
+      new JsonMappingExceptionMapper();
+
+  @Test
+  public void testJsonMappingExceptionMapper() {
+    JsonParser mockParser = Mockito.mock(JsonParser.class);
+    Response response =
+        jsonMappingExceptionMapper.toResponse(JsonMappingException.from(mockParser, ""));
```

Comment:
I think we should use `JerseyTest` to make sure the Exception works in the framework, not only test the function. For example, you can add tests in `com.datastrato.gravitino.server.web.rest.TestMetalakeOperations` to test that.

Reply:
I agree. However, based on the situation with the gravitino, I have two questions
1. Since all the operations related to invalid Json would generate these errors, including but not limited to creating and metalake, so should we also add tests in other `JerseyTest`? , e,g, `TestCatalogOperations`
2. Should we keep these function test(TestXXXMapper)? To me, these tests are unit tests, and what you mentioned is like Integration test?

Reply:
> 1. Since all the operations related to invalid Json would generate these errors, including but not limited to creating and metalake, so should we also add tests in other `JerseyTest`? , e,g, `TestCatalogOperations`

Not need, because they have the same framework entrance. you can only add test in `TestMetalakeOperations`.

> 2.Should we keep these function test(TestXXXMapper)? To me, these tests are unit tests, and what you mentioned is more like a Integration test?

yeah, you can keep them

Reply:
I have updated the patch, please tell me if there are any problems